search-update.rst

```diff
@@ -0,0 +1,90 @@
+.. post:: July 15, 2019
```

Comment:
Also, I think something like `sphinx-search-as-you-type.html` is probably a better URL than `search-update`.

Reply:
Okay.
Do I need to change the blog title also?